client/my-sites/customer-home/controller.jsx

```diff
@@ -28,7 +27,7 @@ export default function( context, next ) {
 
 export function maybeRedirect( context, next ) {
 	const slug = getSelectedSiteSlug( context.store.getState() );
-	if ( ! isEnabled( 'customer-home' ) || 'hide' === abtest( 'customerHomePage' ) ) {
+	if ( 'hide' === abtest( 'customerHomePage' ) ) {
```

Comment:
Should this also redirect away if the user doesn't have the right permissions? (like `canCurrentUserUseCustomerHome()`)

Reply:
No. In such case, we display content that tells the user that they don't have permission to see this page. We settled on this to avoid redirecting users without telling them nothing, so they don't end up in a page they didn't meant to go.